doc/en/fixture.rst

```diff
@@ -804,48 +804,61 @@ without declaring a function argument explicitly or a `usefixtures`_ decorator.
 As a practical example, suppose we have a database fixture which has a
 begin/rollback/commit architecture and we want to automatically surround
 each test method by a transaction and a rollback.  Here is a dummy
-self-contained implementation of this idea::
+self-contained implementation of this idea in a style resembling xUnit setup
+and tearDown methods::
 
-    # content of test_db_transact.py
+    # content of test_foo.py
 
     import pytest
+    import MyFoo
 
-    class DB(object):
-        def __init__(self):
-            self.intransaction = []
-        def begin(self, name):
-            self.intransaction.append(name)
-        def rollback(self):
-            self.intransaction.pop()
+    foo = MyFoo()
 
-    @pytest.fixture(scope="module")
-    def db():
-        return DB()
+    class TestFoo:
```

Comment:
@flub you mean using `setUpClass` instead of a `pytest.fixture` here? If so I agree. 👍